src/redux/books/books.js

```diff
@@ -0,0 +1,43 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import BookApi from '../../service/BooksService';
+
+const ADD_BOOK = 'books/ADD_BOOK';
+const REMOVE_BOOK = 'books/REMOVE_BOOK';
+const LOAD_BOOKS = 'books/LOAD_BOOKS';
+
+export const loadBooks = createAsyncThunk(LOAD_BOOKS, async () => {
+  try {
+    const res = await BookApi.getBookApi();
+    return res.data;
+  } catch (error) {
+    throw new Error(error);
+  }
+});
+
+export const addBook = createAsyncThunk(ADD_BOOK, async (payload, thunkAPI) => {
```

Comment:
**It is commendable that you have implemented all the feedback given by the previous reviewer.** 

Unfortunately, it appears that an error (400) is generated when attempting to post a new book using the link https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/pw2l4oEIk9qbS2aPvGfq/books. 

- [ ] Please address this issue before submitting this Pull Request for approval. Additionally, please ensure all necessary features are functioning properly for successful implementation.